data/json/items/obsolete.json

```diff
@@ -1259,5 +1259,61 @@
     "dispersion": 0,
     "effects": [ "NEVER_MISFIRES", "NON_FOULING" ],
     "relative": { "range": 10, "damage": { "damage_type": "stab", "amount": -5, "armor_penetration": 15 } }
+  },
+  {
+    "id": "steel_scissors",
```

Comment:
Why are you both migrating these and making them obsolete?

Reply:
Last time I tried migrating an item without making it obsolete, which I believe was with the vehicle tracker, the game threw errors at me. Ever since then I've been obsoleting _and_ migrating every item I migrate. In the end, I never got an answer on the dev discord as to whether this should or should not happen.

Reply:
Items that are migrated are replaced with the item they are migrated to, obsoleting them just adds dead data. Whatever you ran into with that sounds like an oversight or bug.

Reply:
It won't hurt to try and see what happens if I do it without the obsoletion. If I get no errors I'll just remove them from the obsoletion file, if I get any I'll file an issue.

Reply:
Yep, @anothersimulacrum the errors still live
![obraz](https://user-images.githubusercontent.com/68503002/220440230-6af93f58-c560-4fec-8f51-b978920bf5a2.png)

Reply:
That's a different error. See my new review comments.